libindy/src/api/anoncreds.rs

```diff
@@ -607,43 +607,58 @@ pub extern fn indy_prover_get_claims_for_proof_req(command_handle: i32,
-///         "requested_predicate_2_uuid": <predicate_info>,
+///         "requested_attr1_referent": <attr_info>,
+///         "requested_attr2_referent": <attr_info>,
+///         "requested_attr3_referent": <attr_info>,
+///         "requested_predicate_1_referent": <predicate_info>,
+///         "requested_predicate_2_referent": <predicate_info>,
 ///     }
 /// requested_claims_json: either a claim or self-attested attribute for each requested attribute
 ///     {
-///         "requested_attr1_uuid": [claim1_uuid_in_wallet, true <reveal_attr>],
-///         "requested_attr2_uuid": [self_attested_attribute],
-///         "requested_attr3_uuid": [claim2_seq_no_in_wallet, false]
-///         "requested_attr4_uuid": [claim2_seq_no_in_wallet, true]
-///         "requested_predicate_1_uuid": [claim2_seq_no_in_wallet],
-///         "requested_predicate_2_uuid": [claim3_seq_no_in_wallet],
+///         "requested_attr1_referent": [claim1_referent_in_wallet, true <reveal_attr>],
+///         "requested_attr2_referent": [self_attested_attribute],
+///         "requested_attr3_referent": [claim2_seq_no_in_wallet, false]
+///         "requested_attr4_referent": [claim2_seq_no_in_wallet, true]
+///         "requested_predicate_1_referent": [claim2_seq_no_in_wallet],
+///         "requested_predicate_2_referent": [claim3_seq_no_in_wallet],
 ///     }
 /// schemas_jsons: all schema jsons participating in the proof request
 ///     {
-///         "claim1_uuid_in_wallet": <schema1>,
-///         "claim2_uuid_in_wallet": <schema2>,
-///         "claim3_uuid_in_wallet": <schema3>,
+///         "claim1_referent_in_wallet": <schema1>,
+///         "claim2_referent_in_wallet": <schema2>,
+///         "claim3_referent_in_wallet": <schema3>,
 ///     }
 ///
 /// master_secret_name: the name of the master secret stored in the wallet
 /// claim_def_jsons: all claim definition jsons participating in the proof request
 ///     {
-///         "claim1_uuid_in_wallet": <claim_def1>,
-///         "claim2_uuid_in_wallet": <claim_def2>,
-///         "claim3_uuid_in_wallet": <claim_def3>,
+///         "claim1_referent_in_wallet": <claim_def1>,
+///         "claim2_referent_in_wallet": <claim_def2>,
+///         "claim3_referent_in_wallet": <claim_def3>,
 ///     }
 /// revoc_regs_jsons: all revocation registry jsons participating in the proof request
 ///     {
-///         "claim1_uuid_in_wallet": <revoc_reg1>,
-///         "claim2_uuid_in_wallet": <revoc_reg2>,
-///         "claim3_uuid_in_wallet": <revoc_reg3>,
+///         "claim1_referent_in_wallet": <revoc_reg1>,
+///         "claim2_referent_in_wallet": <revoc_reg2>,
+///         "claim3_referent_in_wallet": <revoc_reg3>,
 ///     }
 /// cb: Callback that takes command result as parameter.
 ///
+/// where attr_info:
+///     {
+///         "name": attribute name,
+///         "schemas_seq_no": a list of schema sequence numbers allowed for proving of the attribute
```

Comment:
Probably it's better to specify a list of issuer DIDs for **each** schema_seq_no?